integration-tests/otlp/src/main/java/io/opentelemetry/integrationtest/OtlpExporterIntegrationTest.java

```diff
@@ -113,17 +114,22 @@ abstract class OtlpExporterIntegrationTest {
           .put(ResourceAttributes.SERVICE_NAME, "integration test")
           .build();
 
-  @RegisterExtension
-  static final SelfSignedCertificateExtension serverTls = new SelfSignedCertificateExtension();
-
-  @RegisterExtension
-  static final SelfSignedCertificateExtension clientTls = new SelfSignedCertificateExtension();
+  static SelfSignedCertificate serverTls;
+  static SelfSignedCertificate clientTls;
 
   private static OtlpGrpcServer grpcServer;
   private static GenericContainer<?> collector;
 
   @BeforeAll
-  static void beforeAll() {
+  static void beforeAll() throws CertificateException {
+    serverTls = new SelfSignedCertificate();
+    clientTls =
+        new SelfSignedCertificate(
+            Date.from(Instant.now().minusSeconds(365 * 24 * 3600)),
+            Date.from(Instant.ofEpochMilli(253402300799000L)),
+            "EC",
+            256);
```

Comment:
Will revert this file before merging, but just wanted to verify end to end that an EC key works. This test duplicates coverage for the new logic in `TestUtil`. While its nice to see e2e mTLS with different algorithms, its an expensive test to run so we'll want to limit to the happy path using RSA keys.